src/SIM/SIMoutput.C

```diff
@@ -393,9 +396,10 @@ bool SIMoutput::writeGlvBC (int& nBlock, int iStep) const
       continue; // nothing on this patch
 
     if (msgLevel > 1)
-      IFEM::cout <<"Writing boundary conditions for patch "<< i+1 << std::endl;
+      IFEM::cout <<"Writing boundary conditions for patch "
+                 << pch->idx+1 << std::endl;
```

Comment:
isnt' pch->idx 1 based?

Reply:
Obviously, I thought here it was zero-based. But now you made me unsure, so I check again:

It is assigned in the ASMs[123]D::readPatch() methods, e.g.
ASMs3D.C:718       `pch->idx = myModel.size();`

This happens _before_ that patch is added to myModel (which is done by the caller). So then it must be zero-based as the myModel is empty when reading the first patch. Also all other uses of pch->idx indicates it is zero-based.

Reply:
aha, i missed that it happened before. then it's obviously fine.